exploitation/spraying/suspicious-possible-ntfscontrolfile-call.yml

```diff
@@ -0,0 +1,29 @@
+rule:
+  meta:
+    name: suspicious possible NtFsControlFile call
```

Comment:
Let's update this name to more closely follow our naming guidelines: https://github.com/mandiant/capa-rules/blob/master/doc/format.md#rule-name